src/Forms/HTMLEditor/HTMLEditorSanitiser.php

```diff
@@ -17,8 +18,22 @@
  */
 class HTMLEditorSanitiser
 {
+    use Configurable;
     use Injectable;
 
+    /**
+     * rel attribute to add to link elements which have a target attribute (usually "_blank")
+     * this is to done to prevent reverse tabnabbing - see https://www.owasp.org/index.php/Reverse_Tabnabbing
+     * noopener includes the behaviour we want, though some browsers don't yet support it and rely
+     * upon using noreferrer instead - see https://caniuse.com/rel-noopener for current browser compatibility
+     *
+     * @var string
+     */
+    private static $link_noopener_value = 'noopener, noreferrer';
```

Comment:
I believe the value of `rel` attribute should have same rules as `class` or other attributes by separating multiple values with space ` ` instead of comma `, `

I found multiple examples on stackoverflow but mainly it is written in w3.org spec:
```
The attribute, if specified, must have a value that is a set of space-separated tokens representing the various classes that the element belongs to.
```
what is set of space-separated tokens:
https://www.w3.org/TR/2011/WD-html5-20110525/common-microsyntaxes.html#set-of-space-separated-tokens

resource: https://www.w3.org/TR/2011/WD-html5-20110525/elements.html#attributes

Reply:
cheers for pointing out.  have removed the comma

Reply:
Nice catch @jDolba :)